packages/curve-ui-kit/src/shared/ui/DataTable/TableFilters.tsx

```diff
@@ -64,7 +67,7 @@ export const TableFilters = <ColumnIds extends string>({
           </Grid>
         </Fade>
         <Grid
-          size={{ mobile: isExpandedOrValue ? 12 : 'auto', tablet: 6 }}
+          size={{ mobile: hasSearchBar && isExpandedOrValue ? 12 : 'auto', tablet: 6 }}
```

Comment:
`hasSearchBar` is already checked
```suggestion
          size={{ mobile: isExpandedOrValue ? 12 : 'auto', tablet: 6 }}
```